CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * `+[RLMSchema dynamicSchemaForRealm:]` now respects search indexes.
 * `+[RLMProperty isEqualToProperty:]` now checks for equal `indexed` properties.
+* Migration blocks are no longer called when a Realm file is first created.
```

Comment:
I think we actually want to put this in breaking changes. Some users may have relied on the previous behavior.